azure-pipelines.yml

```diff
@@ -210,3 +210,14 @@ jobs:
       testResultsFiles: '**/test-*.xml'
       failTaskOnFailedTests: true
       testRunTitle: 'Publish test results for Python $(PYTHON_VERSION) $(BITS)-bit $(TEST_MODE) Windows'
+
+- job: Linux_PyPy3
+  pool:
+    vmIMage: 'ubuntu-16.04'
+  steps:
+  - script: source tools/pypy-test.sh
+    displayName: 'Run PyPy3 Build / Tests'
```

Comment:
let's add `continueOnError: true` so that failures can be nicely summarized with the next publication step

This is a recent change in our Azure CI, so probably wasn't in the example code before

Reply:
ok

Reply:
Either `failTaskOnFailedTests: true` or `continueOnError: true` is causing CI to pass even though [testing returns 1](https://dev.azure.com/numpy/numpy/_build/results?buildId=3018&view=logs&jobId=b18cf24a-3d78-5967-8f9a-813d25a10c78)

Reply:
They both conspire to allow that to happen--any kind of error in the build / test stage is ignored apart from a yellow warning marker & continuation to the next Azure stage is allowed.

The Azure job will only fail at the test publication stage if the test result files contain a report of a failed test, but the linear algebra backend error didn't prevent the tests from passing.

Maybe there's a way to improve granularity even more there, but maybe we should just focus on having the PyPy job consistent with the other Azure entries in this PR, which I think it is now.